wrapper.js

```diff
@@ -31,12 +31,30 @@ function setupMethods (soljson) {
     };
   }
 
+  var alloc;
+  if ('_solidity_alloc' in soljson) {
+    alloc = soljson.cwrap('solidity_alloc', 'number', [ 'number' ]);
+  } else {
+    // pre 0.6.0
+    alloc = soljson._malloc;
```

Comment:
Maybe assert that it exists (if applicable)?

Reply:
Added.